web-ts/package.json

```diff
@@ -0,0 +1,150 @@
+{
+  "name": "web-ts",
+  "version": "0.1.0",
+  "private": true,
+  "engines": {
+    "node": ">=6.9",
+    "npm": ">=3.8"
+  },
+  "scripts": {
+    "start": "cross-env TS_NODE_PROJECT=\"tools/tsconfig.webpack.json\" webpack-dev-server --config tools/webpack.config.ts --progress --env.config_name=integreat-test-cms --env.dev_server",
+    "start:malte": "cross-env TS_NODE_PROJECT=\"tools/tsconfig.webpack.json\" webpack-dev-server --config tools/webpack.config.ts --progress --env.config_name=malte --env.dev_server",
+    "start:aschaffenburg": "cross-env TS_NODE_PROJECT=\"tools/tsconfig.webpack.json\" webpack-dev-server --config tools/webpack.config.ts --progress --env.config_name=aschaffenburg --env.dev_server",
+    "build": "yarn build:integreat",
+    "build:integreat": "cross-env TS_NODE_PROJECT=\"tools/tsconfig.webpack.json\" webpack --config tools/webpack.config.ts --env.config_name=integreat",
+    "build:malte": "cross-env TS_NODE_PROJECT=\"tools/tsconfig.webpack.json\" webpack --config tools/webpack.config.ts --env.config_name=malte",
+    "build:aschaffenburg": "cross-env TS_NODE_PROJECT=\"tools/tsconfig.webpack.json\" webpack --config tools/webpack.config.ts --env.config_name=aschaffenburg",
+    "build:integreat-test-cms": "cross-env TS_NODE_PROJECT=\"tools/tsconfig.webpack.json\" webpack --config tools/webpack.config.ts --env.config_name=integreat-test-cms",
+    "check:build": "yarn es-check es2020 ./dist/**/*.js --verbose",
+    "test": "jest --config jest.config.js",
+    "test:coverage": "yarn run test --coverage",
+    "test:ci": "yarn run test:coverage --ci",
+    "test:watch": "yarn run test --watchAll",
+    "test:update": "yarn run test -u",
+    "lint": "yarn run eslint && yarn run stylelint",
+    "lint:ci": "yarn run eslint --quiet --format junit -o reports/lint/junit-lint.xml && yarn run stylelint",
+    "eslint:fix": "eslint . --fix",
+    "eslint": "eslint .",
+    "stylelint": "stylelint --config stylelint.config.js \"src/**/*.{js,css}\""
+  },
+  "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.34",
+    "@fortawesome/free-regular-svg-icons": "^5.15.2",
+    "@fortawesome/free-solid-svg-icons": "^5.15.2",
+    "@fortawesome/react-fontawesome": "^0.1.14",
+    "@integreat-app/react-sticky-headroom": "^1.1.0",
+    "@sentry/react": "6.2.1",
+    "@types/jest": "^26.0.15",
+    "@types/node": "^12.0.0",
+    "@types/react": "^17.0.0",
+    "@types/react-dom": "^17.0.0",
```

Comment:
This was part of `IGAPP-web-ts-init`. Afaik the types are normally devDependencies. Or am I wrong here?

Reply:
You can either fix this in this issue or we do this later. Also created an issue to remove unused libraries: https://issues.tuerantuer.org/browse/IGAPP-622